### DISCUSSION TITLE
Combine renovate prs

### DISCUSSION THREAD

**Discussion by icecream17** [renovatebot/renovate#14468] [Request Help] [answered]
Is there a way to combine https://github.com/icecream17/ts-python-chess/pull/247 and https://github.com/icecream17/ts-python-chess/pull/244?

**Comment by rarkins** [comment, marked as answer]
```
"packageRules": [{
  "matchPackagePatterns": ["eslint"],
  "groupName": "eslint"
}]
```

This will capture everything with `eslint` in its name.
